packages/component-library/stories/StackedAreaChart.story.js

```diff
@@ -104,87 +69,227 @@ const customLegend = legendData => {
-      const subtitle = text("Subtitle", sampleSubtitle);
-      const title = text("Title", sampleTitle);
-      const xLabel = text("xLabel", sampleXLabel);
-      const yLabel = text("yLabel", sampleYLabel);
+      const data = object("Data", sampleEnrollmentData, GROUP_IDS.DATA);
 
       return (
         <StackedAreaChart
           data={data}
           dataKey={dataKey}
-          dataKeyLabel={dataKeyLabel}
           dataValue={dataValue}
-          dataValueLabel={dataValueLabel}
           dataSeries={dataSeries}
           dataSeriesLabel={dataSeriesLabel}
-          domain={domain}
-          size={size}
           subtitle={subtitle}
           title={title}
           xLabel={xLabel}
           yLabel={yLabel}
-          legendComponent={customLegend}
+          xNumberFormatter={x => civicFormat[optionSelectX](x)}
+          yNumberFormatter={y => civicFormat[optionSelectY](y)}
         />
       );
-    });
+    })
+    .add(
+      "Custom",
+      () => {
+        const sampleEnrollmentData = [
+          { year: 2005, students: 34, group: "BAA" },
+          { year: 2006, students: 28, group: "BAA" },
+          { year: 2007, students: 17, group: "BAA" },
+          { year: 2008, students: 18, group: "BAA" },
+          { year: 2009, students: 14, group: "BAA" },
+          { year: 2010, students: 23, group: "BAA" },
+          { year: 2011, students: 21, group: "BAA" },
+          { year: 2012, students: 17, group: "BAA" },
+          { year: 2013, students: 19, group: "BAA" },
+          { year: 2014, students: 16, group: "BAA" },
+
+          { year: 2005, students: 34, group: "HIS" },
+          { year: 2006, students: 28, group: "HIS" },
+          { year: 2007, students: 24, group: "HIS" },
+          { year: 2008, students: 24, group: "HIS" },
+          { year: 2009, students: 37, group: "HIS" },
+          { year: 2010, students: 37, group: "HIS" },
+          { year: 2011, students: 44, group: "HIS" },
+          { year: 2012, students: 42, group: "HIS" },
+          { year: 2013, students: 48, group: "HIS" },
+          { year: 2014, students: 45, group: "HIS" },
+
+          { year: 2005, students: 0, group: "MULTI" },
+          { year: 2006, students: 0, group: "MULTI" },
+          { year: 2007, students: 28, group: "MULTI" },
+          { year: 2008, students: 41, group: "MULTI" },
+          { year: 2009, students: 23, group: "MULTI" },
+          { year: 2010, students: 42, group: "MULTI" },
+          { year: 2011, students: 54, group: "MULTI" },
+          { year: 2012, students: 55, group: "MULTI" },
+          { year: 2013, students: 56, group: "MULTI" },
+          { year: 2014, students: 59, group: "MULTI" }
+        ];
+        const enrollmentDataSeries = "group";
+        const enrollmentDataSeriesLabels = [
+          { category: "BAA", label: "Black/African American" },
+          { category: "HIS", label: "Hispanic/Latino" },
+          { category: "MULTI", label: "Multi-Ethnic" }
+        ];
```

Comment:
If we're going to use this sample data, I'd like to include the source of the demographic group labeling in the subtitle.

For example, demographic categories designated by PPS, or census

Reply:
Great suggestion.  I used a subset of the data in the Students from Historically Underrepresented Groups chart here:  http://civicplatform.org/cities/portland/neighborhood

But, it doesn't give the source of the demographic group labeling here.  Where would I find it?

Reply:
Suggestion: just add "using selected Census categories"